pydantic/colors.py

```diff
@@ -0,0 +1,175 @@
+"""
+Color definitions are  used as per CSS3 specification:
+http://www.w3.org/TR/css3-color/#svg-color
+
+In turn CSS3 is based on SVG specification for color names:
+http://www.w3.org/TR/SVG11/types.html#ColorKeywords
+"""
+
+
+# {name: (hex, rgb)}
+BY_NAME = {
+    'aliceblue': ('f0f8ff', (240, 248, 255)),
```

Comment:
I don't think we need hex and tuple colours here since once can be calculated from the other.

You can just calculate the other one on model init as you do below.